.github/workflows/codeql.yml

```diff
@@ -25,13 +25,11 @@ jobs:
 
     steps:
     - name: Checkout repository
-      # uses: actions/checkout@v3
-      uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      # uses: github/codeql-action/init@v2
-      uses: github/codeql-action/init@c3b6fce4ee2ca25bc1066aa3bf73962fda0e8898
+      uses: github/codeql-action/init@b8efe4dc6ab6d31abe3ec159420d2a4916880800
```

Comment:
Trying to understand why you picked those specific commit ids. It's in between these two tags. Should we not pick the latest tagged version?
https://github.com/github/codeql-action/commits/v3.26.5
https://github.com/github/codeql-action/commits/v3.26.6